### DISCUSSION TITLE
How to generate SDF files when running ORFS

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD-flow-scripts#1438] [Q&A] [answered]
I do see that `OpenSTA` had a command called `write_sdf` which should be used to generate SDF files. However I noticed that no SDF is generated during the flow run (I do see `*.spef` files and other files being generated). Is there a reason why the flow does not automatically write out SDF files? What would be the recommended procedure to get these files generated?  Given that the script directory does not contain any `OpenSTA` specific TCL file I am not sure I should put the `write_sdf` directive.

**Comment by maliberty** [comment]
It wouldn't be hard to add but there is no purpose for such a file in the current flow.  What is your use for the SDF?

**Reply by gkamendje** [reply]
I was thinking to use it for back annotated Gate Level Simulations. Would you have any suggestion on how to achieve this in the current flow without SDF file?

**Reply by maliberty** [reply, marked as answer]
The easiest would be just to add the command to scripts/final_report.tcl

**Reply by gkamendje** [reply]
Thanks for the hint. That make sense
